parse/parser.go

```diff
@@ -280,10 +280,39 @@ func parseReturnStatement(buf TokenBuffer) (ast.Expression, []error) {
 
 // TODO: implement me w/ test cases :-)
 func parseAssignStatement(buf TokenBuffer) (*ast.AssignStatement, []error) {
-	return nil, nil
+	stmt := &ast.AssignStatement{}
+
+	tok := buf.Read()
+	if !isDataStructure(tok) {
```

Comment:
This is just a question.
Can koa make user-defined data structure? (like struct or class?)

Reply:
We will!

Reply:
Cool ~3~